src/js/components/award/shared/awardAmountsSection/charts/SharedBarComponents.jsx

```diff
@@ -11,7 +11,7 @@ const BarValue = ({
     number
 }) => (
     <div
-        style={style}
+        style={{ ...style, minWidth: '100px' }}
```

Comment:
Recommend adding a comment that this is to allow enough width for label text (we are not manipulating the line itself, which could be misleading about the data)